CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 #
-# Copyright (c) 2014 LAAS-CNRS
-# Authors: Mathieu Geisert
+# Copyright (c) 2014, 2020 LAAS-CNRS
+# Authors: Mathieu Geisert, Guilhem Saurel
```

Comment:
Well, since here, you may also add my name :)

Reply:
Of course :)